src/rdiff_backup/log.py

```diff
@@ -164,12 +156,12 @@ def log_to_file(self, message):
     def log_to_term(self, message, verbosity):
         """Write message to stdout/stderr"""
         if verbosity <= 2 or Globals.server:
-            termfp = sys.stderr
+            termfp = sys.stderr.buffer
         else:
-            termfp = sys.stdout
+            termfp = sys.stdout.buffer
```

Comment:
I think, it would be a better approach to rollback the changes in `log_to_term` and only set `sys.stdout.reconfigure(errors='backslashreplace')` in the `else:` part of the code (`sys.stderr.errors` is by default `backslashreplace'). It would actually be even better to place it in a part of the code which is only run once (perhaps in the ```__init__``` of the object?).

Why is it better (IMHO always)? Because 99% of the log messages are already strings, so this limits the amount of conversions necessary.

I would definitely use `backslashreplace` to be consistent between stderr, stdout and the logfile.

Reply:
Didn't know it was possible to reconfigure the encoding of the stdout. This is useful !
Will fix this.

Reply:
Bon. Forget the reconfigure it's only available starting with Python3.7. 
![image](https://user-images.githubusercontent.com/921251/82664953-cd81f280-9c00-11ea-96ce-5c4b18d6b503.png)

Reply:
Ah, shoot! Then let's go with your approach, but with `backslashreplace` for consistency, unless you have a reason for `replace` instead.

Reply:
For now I'm keeping the current implementation unless you have another recommendation ? I cannot do `sys.stdout.errors = 'backslashreplace'`. Using the buffer directly seams to be the most simple way to manage this.

Reply:
Perfect. I will change it for `backslashreplace`.